VK/VK/Presentation/Friends/Models/Birthdays.swift

```diff
@@ -0,0 +1,14 @@
+//
+//  Birthdays.swift
+//  VK
+//
+//  Created by Алена Панченко on 05.11.2022.
+//
+
+import Foundation
+
+/// Модель дней рождения
+struct Birthdays {
```

Comment:
```suggestion
struct Birthday {
```
модель в единственном числе

Reply:
и почему свойства не let?